core/src/main/kotlin/behavior/channel/StageChannelBehavior.kt

```diff
@@ -69,6 +73,20 @@ suspend fun StageChannelBehavior.edit(builder: StageVoiceChannelModifyBuilder.()
     return Channel.from(data, kord) as StageChannel
 }
 
+suspend fun StageChannelBehavior.createStageInstance(topic: String): StageInstance {
+    val instance = kord.rest.stageInstance.createStageInstance(id, topic)
+    val data = StageInstanceData.from(instance)
+
+    return StageInstance(data, kord, supplier)
+}
+
+suspend fun StageChannelBehavior.getStageInstance(): StageInstance {
+    val instance = kord.rest.stageInstance.getStageInstance(id)
+    val data = StageInstanceData.from(instance)
+
+    return StageInstance(data, kord, supplier)
+}
+
```

Comment:
These shouldn't be extension functions. Generally online inline functions are added as extensions for interfaces.